spec/controllers/documents_controller_spec.rb

```diff
@@ -76,9 +76,9 @@
 
     it "should render documents with long descriptions properly" do
 
-      response.body.should have_css('.wiki p')
-      response.body.should have_css('.wiki p', text: (document.description.split("\n").first + '...'))
-      response.body.should have_css('.wiki p', text: /EndOfLineHere.../)
+      expect(response.body).to have_css('.wiki p')
+      expect(response.body).to have_css('.wiki p', text: (document.description.split("\n").first + '...'))
```

Comment:
Line is too long. [106/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.